src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2814,6 +2858,20 @@ components:
           description: Date from which to remove older log files from repository.
           nullable: true
           example: "2022-07-16"
+    AddMember:
+      description: UserMember options
+      type: object
+      properties:
+        userId:
+          type: integer
+          description: User id of the new member.
+          nullable: true
+          example: 2
+        perm:
+          type: integer
+          description: Default permission of the new member.
+          nullable: true
```

Comment:
```suggestion
          nullable: false
          minimum: 0
          maximum: 2
```

Reply:
Please add the `minimum` and `maximum` as well to the `AddMember.perm` model

Reply:
done